### ISSUE TITLE
Update Simulator for 3Node

### ISSUE THREAD

**Issue by Mik-TF** [threefoldtech/tfgrid-sdk-ts#1830]
## Note

This issue is linked with issue #133 but some new things are asked so I created a new issue

## Request

[Robert asked on a ThreeFold forum post](https://forum.threefold.io/t/issues-that-need-to-be-resolved-before-new-strategy/4181) to update the simulator.

•  The name Titan needs to be changed to ‘Certified Nodes’;
•  At this certified section the option of HDD need to be added, since certifies nodes can have HDD which also generates extra TFT;
•  GPU needs to be added to both DYI and certified nodes with the option to manually set a price
•  For V4, the calculator should provide utilization cost defaults so farmers can estimate their profits (how much TFT for x cpu, x mem and x storage);

**Comment by ramezsaeed** [comment]
Blocked on KDS review

**Comment by mik-tf** [comment]
I'd be happy to help for this issue. We have frequent requests by farmers to update this.
